PowerEditor/installer/nativeLang/vietnamese.xml

```diff
@@ -1479,13 +1594,19 @@ Tìm trong tất cả các tệp ngoại trừ exe, obj &amp;&amp; log:
             <find-result-line-prefix value="Dòng"/> <!-- Must not begin with space or tab character -->
             <find-regex-zero-length-match value="phần khớp có độ dài là không" />
             <session-save-folder-as-workspace value="Lưu thư mục như là không gian làm việc" />
-            <tab-untitled-string value="new " />
+            <tab-untitled-string value="mới " />
             <file-save-assign-type value="&amp;Thêm phần mở rộng vào sau" />
             <close-panel-tip value="Đóng" />
             <IncrementalFind-FSFound value="$INT_REPLACE$ kết quả so khớp" />
             <IncrementalFind-FSNotFound value="Không tìm thấy cụm từ" />
             <IncrementalFind-FSTopReached value="Đã đi tới đầu trang, tiếp tục từ bên dưới" />
             <IncrementalFind-FSEndReached value="Đã đi tới cuối trang, tiếp tục từ bên trên" />
+            <contextMenu-styleAlloccurrencesOfToken value="Đặt kiểu dáng tất cả trường hợp của từ tố" />
+            <contextMenu-styleOneToken value="Đặt kiểu dáng một từ tố" />
+            <contextMenu-clearStyle value="Dọn sạch kiểu dáng" />
+            <enable-disable-largeFileRestriction-tip value="Bật tắt &quot;Bật giới hạn tệp lớn&quot; chỉ có tác dụng sau khi tắt đi mở lại tệp lớn đó" />
+            <change-largeFileRestriction_fileLength-tip value="Chỉnh sửa giá trị kích cỡ tệp chỉ có tác dụng sau khi tắt đi mở lại tệp lớn đó" />
+            <largeFileRestriction_wordWrap-tip value="Bật tuỳ chọn &quot;Vô hiệu hoá bọc từ ở mọi nơi&quot; sẽ tắt Bọc từ đối với tất cả tài liệu được mở sau khi mở một tệp lớn. Bạn có thể bật lại tính năng Bọc từ một cách thủ công." />
```

Comment:
Could you please update the PR to the latest updated english.xml? 
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/12404

Reply:
I have updated the translation to the latest english.xml.